frontend/src/scenes/userLogic.ts

```diff
@@ -241,10 +248,10 @@ export const userLogic = kea<userLogicType>([
                     : [],
         ],
 
-        themeMode: [
-            (s) => [s.user],
-            (user): UserTheme => {
-                return user?.theme_mode || 'light'
+        userThemeMode: [
+            (s) => [s.user, s.themeFromCookie],
+            (user, themeFromCookie): UserTheme => {
+                return user?.theme_mode || themeFromCookie || 'light'
```

Comment:
should be decently easy to set up a flag here I believe. Let's test it because we can, and because colors can have a big impact on signup.

Reply:
@Twixes how could we feature flag this just for the auth pages? I can do it if you'd like just needs some hints on where to start.

Reply:
There's this [setFeatureFlags](https://github.com/PostHog/posthog/blob/master/frontend/src/mocks/browser.tsx#L34) utility. 

Example usage: https://github.com/PostHog/posthog/blob/master/frontend/src/layout/FeaturePreviews/FeaturePreviews.stories.tsx#L33

I haven't actually done this myself so this could be the wrong direction.